mass_mailing_custom_unsubscribe/migrations/8.0.2.0.0/pre-migrate.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not, if somebody upgrades to v9 from a version previous than 8.0.2.0.0, they'll need this.